### PR TITLE
[DDW-261] Shelley FF features

### PR DESCRIPTION
**Pull request by nikolaglumac**
This PR prepares all of the Shelley FF features for the Daedalus 1.0.0-STN1 release.

## Todos

- [x] Update text copy and Japanese translations

### English

Screenshot 2020-06-16 at 11 55 05
Screenshot 2020-06-16 at 11 55 16
Screenshot 2020-06-16 at 11 55 24

### Japanese

Screenshot 2020-06-16 at 11 55 31

Screenshot 2020-06-16 at 11 55 36

Screenshot 2020-06-16 at 11 55 40

---

## Testing Checklist

- [Slack QA thread](https://input-output-rnd.slack.com/messages/GGKFXSKC6)
- [ ] Test

---

### Basics

- [ ] PR has been assigned and has appropriate labels (`feature`/`bug`/`chore`, `release-x.x.x`)
- [ ] PR is updated to the most recent version of the target branch (and there are no conflicts)
- [ ] PR has a good description that summarizes all changes
- [ ] PR has default-sized Daedalus window screenshots or animated GIFs of important UI changes:
  - [ ] In English
  - [ ] In Japanese
- [ ] CHANGELOG entry has been added to the top of the appropriate section (*Features*, *Fixes*, *Chores*) and is linked to the correct PR on GitHub
- [ ] Automated tests: All acceptance and unit tests are passing (`yarn test`)
- [ ] Manual tests (minimum tests should cover newly added feature/fix): App works correctly in *development* build (`yarn dev`)
- [ ] Manual tests (minimum tests should cover newly added feature/fix): App works correctly in *production* build (`yarn package` / CI builds)
- [ ] There are no *flow* errors or warnings (`yarn flow:test`)
- [ ] There are no *lint* errors or warnings (`yarn lint`)
- [ ] There are no *prettier* errors or warnings (`yarn prettier:check`)
- [ ] There are no missing translations (running `yarn manage:translations` produces no changes)
- [ ] Text changes are proofread and approved (Jane Wild / Amy Reeve)
- [ ] Japanese text changes are proofread and approved (Junko Oda)
- [ ] UI changes look good in all themes (Alexander Rukin)
- [ ] Storybook works and no stories are broken (`yarn storybook`)
- [ ] In case of dependency changes `yarn.lock` file is updated

### Code Quality
- [ ] Important parts of the code are properly commented and documented
- [ ] Code is properly typed with flow
- [ ] React components are split-up enough to avoid unnecessary re-renderings
- [ ] Any code that only works in main process is neatly separated from components

### Testing
- [ ] New feature/change is covered by acceptance tests
- [ ] New feature/change is manually tested and approved by QA team
- [ ] All existing acceptance tests are still up-to-date
- [ ] New feature/change is covered by Daedalus Testing scenario
- [ ] All existing Daedalus Testing scenarios are still up-to-date

### After Review
- [ ] Merge the PR
- [ ] Delete the source branch
- [ ] Move the ticket to `done` column on the YouTrack board
- [ ] Update Slack QA thread by marking it with a green checkmark

### CODE REVIEW

**Comment by ManusMcCole** [comment]
The newest build has reverted back to the text copy and header from the Incentivised Testnet on Windows. Checking Mac now 
image

**Comment by nikolaglumac** [comment]
> The newest build has reverted back to the text copy and header from the Incentivised Testnet on Windows. Checking Mac now
> image

@ManusMcCole you are using a wrong build... Please use this one: https://buildkite.com/input-output-hk/daedalus/builds/13186

**Comment by ManusMcCole** [comment]
Sorry I just realised and deleted. Got build 13168 mixed up with 13186 look very similar

**Comment by gnpf** [comment]
@nikolaglumac The path for the state directory in Linux is still **"Daedalus/ff"**.

image

Also in Linux, the name is "Daedalus ff" instead of "Daedalus Shelley Testnet"

image

File: source/renderer/app/i18n/locales/ja-JP.json [hunk 1]
```diff
@@ -818,6 +822,7 @@
   "wallet.transaction.type.exchange": "換金",
   "wallet.transaction.unresolvedInputAddressesAdditionalLabel": "アドレスを表示する。",
   "wallet.transaction.unresolvedInputAddressesLinkLabel": "Cardanoエクスプローラーでこのトランザクションを開き",
+  "wallet.transaction.unresolvedInputAddressesLinkUnavailableLabel": "!!!Cardano Explorer link currently unavailable.",
```
Comment (nikolaglumac) [inline source/renderer/app/i18n/locales/ja-JP.json:825]: @darko-mijic @daniloprates we need to add this translation. I have asked Junko to provide it.

**Comment by nikolaglumac** [comment]
> @nikolaglumac The path for the state directory in Linux is still **"Daedalus/ff"**.
> 
> image
> 
> Also in Linux, the name is "Daedalus ff" instead of "Daedalus Shelley Testnet"
> 
> image

@cleverca22 @disassembler can you help us with this?

File: source/renderer/app/i18n/locales/ja-JP.json (hunk 1 above)
Comment (nikolaglumac) [reply source/renderer/app/i18n/locales/ja-JP.json]: @darko-mijic @daniloprates resolved in https://github.com/input-output-hk/daedalus/pull/2034/commits/3befa418d121bab88573b74afb6b5c6a7774f741

**Comment by nikolaglumac** [comment]
> @nikolaglumac The path for the state directory in Linux is still **"Daedalus/ff"**.
> 
> image
> 
> Also in Linux, the name is "Daedalus ff" instead of "Daedalus Shelley Testnet"
> 
> image

@gnpf please create a new card for this 🙏

**Pull request merged**